database/DevHome.Database/DevHomeDatabaseContext.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.ComponentModel.DataAnnotations.Schema;
+using DevHome.Database.DatabaseModels.RepositoryManagement;
+using Microsoft.EntityFrameworkCore;
+
+namespace DevHome.Database;
+
+public class DevHomeDatabaseContext : DbContext
+{
+    public DbSet<Repository> Repositories { get; set; }
+
+    public DbSet<RepositoryMetadata> RepositoryMetadatas { get; set; }
+
+    public string DbPath { get; }
+
+    public DevHomeDatabaseContext()
+    {
+        // Not the final path.  It will change before going into main.
+        // Needs a configurable location for testing.
+        var folder = Environment.SpecialFolder.LocalApplicationData;
+        var path = Environment.GetFolderPath(folder);
+        DbPath = Path.Join(path, "DevHome.db");
+    }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        // If I have time, this should be split like service extensions.
+        // As more entities are added, the longer this method will get.
```

Comment:
Grammatically incorrect. Consider _As more entities are added, this method will get longer_

Reply:
Changed.